### ISSUE TITLE
Auto-trigger broken?

### ISSUE THREAD

**Issue by bcm0** [msiniscalchi/atom-latextools#140]
Included a non empty .bib file with \bibliography(test) in my .tex document.
When I start to type \cite{ no suggestions will be made. The select view panel won't pop up.
Can you confirm this? How do you debug?

**Comment by edwinksl** [comment]
@nokianhunter For me, it was caused by my choice of the UI theme. Specifically, I was using the Nucleus Dark UI. Changing to a different UI theme fixed the problem.

**Comment by nokianhunter** [comment]
Thanks for the tip.
I changed UI and syntax theme to Atom Dark but it didn't help
Unfortunately I must have something else configured wrong.
I also tried a fresh Atom configuration with only the latextools package added.
Additionally a second .tex/.bib example didn't work.
Do you have other suggestions?

**Comment by edwinksl** [comment]
Could you provide minimal examples of a bibliography and a .tex document where \cite would fail?

**Comment by bcm0** [comment]
I took this example: https://tex.stackexchange.com/questions/225209/bibtex-simple-example
I tried it as it is and in addition like  below:

```
\documentclass{article}
\usepackage[hyperref=true,
    natbib=true,
    style=authortitle-dw,
    backend=biber]{biblatex}

% jobname.bib is in a extra file in the same directory

\addbibresource{jobname.bib}

\begin{document}
    Hallo world
    \cite{einstein}
    \printbibliography
\end{document}

```

Thanks for helping me out

**Comment by ig0774** [comment]
@nokianhunter: Is your `\addbibresource` line actually `\addbibresource{\jobname.bib}`? We currently don't support completions from files using the `\jobname` macro, though it's on the list of things to add.

**Comment by nokianhunter** [comment]
No it's like this "\addbibresource{jobname.bib}"

**Comment by nokianhunter** [comment]
I didn't know \jobname is a macro. But the line is exactly like this "\addbibresource{jobname.bib}".
Does it work in your configuration?
Any idea how to compare?

**Comment by ig0774** [comment]
@nokianhunter Your example works for me. The only thing I can think of is to look at the Atom console window (it's in the developer tools bit of the section). It also might be helpful to see an example entry from your bib file...

**Comment by nokianhunter** [comment]
My bib file looks like this:
```
@artikel{einstein,
  author = {Albert Einstein},
  title = {the true about tree},
  journaltitle = {Annalen der Physik},
  year = {1905},
  volume = {322},
  number = {10},
  pages = {891-921}
}
```

**Comment by nokianhunter** [comment]
Dev console output when choosing "Latextools: Ref Cite Complete":
requiring if needed: completion-manager
requiring if needed: snippet-manager
??

**Comment by ig0774** [comment]
Running the command directly is actually not too useful, unfortunately :). It checks to see if it's inside a citation command before doing anything. It'd be better to see the output when actually typing `{` after the cite command.

**Comment by ig0774** [comment]
@nokianhunter Everything works for me using your sample entry and the sample tex file on 1.12.4.

![citation](https://cloud.githubusercontent.com/assets/212893/20443388/8766c4b6-adc4-11e6-9761-995f022145a9.gif)

**Comment by bcm0** [comment]
Thanks for testing and making this example gif.
Looking at your example, I recognized I had no syntax highlighting. Installing language-latex helped.
And finally the auto-trigger worked again!
Afterwards I found this on the project page:
`
The package does not provide syntax highlighting. Rather, it is supposed to be used in conjunction with the language-latex package: this must be installed for snippets to work.`


The package "keyboard-localization" also prevents the auto-trigger event from happening.
